### ISSUE TITLE
Can't pickup items in areas with Apotheosis Spawners in them (after a period of time).

### ISSUE THREAD

**Issue by DDock** [Shadows-of-Fire/Apotheosis#304]
I am running the Valhelsia 2 modpack and I have seen other users with the same issue on Reddit (linked below). After a period of time you can no longer pickup items on the ground. You can use a "magnetic" type item to bring the items out of the affected area to pick them up. These broken areas seem to be related to an interaction with Apotheosis, mostly the spawners. I looked in the Issues section, but I don't see any issues, closed or open about pickup of items. I have seen areas without obvious spawners, but I've seen random spawners in cave systems that could be causing this in those areas, but can't confirm. I have a blaze farm using spawners that will eventually have this issue, but it seems very random on what triggers the area to stop working correctly.

https://www.curseforge.com/minecraft/modpacks/valhelsia-2
https://www.reddit.com/r/feedthebeast/comments/jmwa3g/valhelsia_2_cant_pick_up_items_in_one_chunk/

I'm willing to work with the mod authors to try to attempt to solve this issue. It has been plaguing my server for awhile now and the only temporary way to fix it is to restart the server. Let me know what I can do to help.

**Comment by TwoPointInfinity** [comment]
Also having this issue. after a bit i cant pick items up, my magnet will drag them out of the specific subchunk and then i pick them up.

**Comment by Shadows-of-Fire** [comment]
How long does this typically take to trigger?  I've had a spawner sitting here for a while now and have had no issue gathering items.  I also cannot see anything that would even make this happen, as the code for the apoth spawner is the exact same as vanilla with a couple additional conditionals for when certain triggers are active (like condition ignorance)

**Comment by TwoPointInfinity** [comment]
I've had it take about 10 minutes but it's also not happened for a few hours. Sometimes i would pick up a some items and not others. I've had spawners at the surface and underground both do this. This happened in valhelsia 2, val 3 and enigmatica 6. We try bringing our mobs out of the area and killing them, but it seems like its not the spawner itself causing the issue, its the apoth spawner mobs. Where ever they die will be the affected chunk/subchunk.

**Comment by DDock** [comment]
Typically if you have your spawner setup to spawn more quickly it tends to break quicker. We have created two blaze spawn rooms, one originally at a friends house, then I created one at mine. I have found if you have it set to fast it breaks the area to pick up items and xp. Sadly even with a reasonable speed spawner (about 1 per second) it can still happen, but seems very random. Like TPI said it could be 10 minutes, or 2 hours. I have even come across chunks that don't work out in the world. I have come to assume they have the cave spawners inside that have broken the area. I don't mind giving you access to my server if you would like to do some poking around. At this time the people that were playing have moved on so the server is really no longer being used. I'm assuming at this point it is an interaction with another mod, but I'm sure you would prefer it to be fixed than leave it causing issues with one of the more popular modpacks currently on CurseForge.
